src/components/Bale/TabletBale.js

```diff
@@ -1,50 +1,74 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { View, Image } from 'react-native';
-import { Text, ListItem } from 'react-native-elements';
+import { View, Image, StyleSheet, Text } from 'react-native';
 import FardoIcon from '../../assets/images/FardoIcon.png';
 import Button from '../common/Button';
 import strings from '../../localization';
-import styles from './styles';
+import Colors from '../../helpers/Colors';
+
+const stylesTabletItem = StyleSheet.create({
```

Comment:
Move the styles to the corresponding styles.js please

Reply:
Moved to style